kernel/Kconfig

```diff
@@ -969,10 +969,18 @@ endmenu
 
 menu "Device Options"
 
-config HAS_DYNAMIC_DEVICE_HANDLES
-	bool
+config DEVICE_DEPS
+	bool "Store device dependencies"
+	help
+	  When enabled, device dependencies will be stored so that they can be
```

Comment:
Would it nicer to be a bit more explicit saying that they will be stored in the image, and that disabling this feature can typically save from a few tenths to a couple hundredth bytes of ROM?

Reply:
Added a comment about ROM (or RAM) usage